constantine/eth_verkle_ipa/barycentric_form.nim

```diff
@@ -21,41 +21,54 @@ import
 
 # Please refer to https://hackmd.io/mJeCRcawTRqr9BooVpHv5g 
 
+
+## newPrecomputedWeights generates the precomputed weights for the barycentric formula
+## Let's say we have 2 arrays of the same length and we join them together
+## This is how we shall be storing A'(x_i) and 1/A'(x_i), this midpoint is used to compute
+## the offset to wherever we need to access the 1/A'(x_i) value.
```

Comment:
In Nim the doc comments should be after the function name and indented or the compiler will not properly document the functions